odolbeau/phone-number-bundle/3.0/post-install.txt

```diff
@@ -0,0 +1,8 @@
+<bg=green;fg=white></>
+<bg=green;fg=white> You now can use the phone bundle as described here: </>
+<bg=green;fg=white> https://github.com/odolbeau/phone-number-bundle#usage </>
+<bg=green;fg=white> The Doctrine configuration has been already done.</>
+<bg=green;fg=white> You now can use `phone_number` as an ORM-Type: </>
+
+  /** @ORM\Column(type="phone_number") */
+  private $phoneNumber;
```

Comment:
you shouldn't need this file

Reply:
It's printed in the console. Which is nice.

Reply:
Maybe but it's kinda useless most of the time, so it's really there if you really have something to say. which is not the case here.

Imagine if you have 10 recipes executed with each of them one message...

Reply:
Not necessarily, with this RFC we should be able to replay the message symfony/flex#518

Reply:
agreed with @Taluu recipes are not documentation.
What if I don't use `phone-number-bundle` for ORM, but for Form, or twig ?